harvester/src/main/java/com/github/reflectoring/infiniboard/harvester/source/url/UrlSourceJob.java

```diff
@@ -56,11 +63,42 @@ protected void executeInternal(ApplicationContext context, JobKey jobKey, Map co
 
             upsertResults(context, jobKey, results);
 
+        } catch (SSLHandshakeException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1166)